en.json

```diff
@@ -377,7 +377,7 @@
         "Undo.BakeMeshes" : "Bake {name}",
         "Undo.ChangeMesh" : "Change Mesh",
         "Undo.InsertParent" : "Insert Parent for {name}",
-        "Undo.Wiggler" : "Evil Stuff",
+        "Undo.Wiggler" : "Wiggler",
```

Comment:
Please don't change this, it's an easter egg! xP

Reply:
I had my suspicions ;)